docs/002-install-redisearch.md

```diff
@@ -5,17 +5,17 @@ You have multiple ways to run RediSearch:
 * building from [sources](https://github.com/RediSearch/RediSearch) and installing it inside an existing Redis Instance
 * using [Redis Cloud](https://redislabs.com/redis-enterprise-cloud/) _(when RediSearch module 2.0 available)_
 * using [Redis Enterprise](https://redislabs.com/redis-enterprise-software/) _(when RediSearch module 2.0 available)_
-* using [Docker](https://hub.docker.com/r/redislabs/redisearch/)
+* using [Docker](https://hub.docker.com/r/redis/redis-stack)
 
 Let's use Docker for now.
 
 **1.1 Open a terminal an run the following command**
 
 
 ```
-> docker run -it --rm --name redis-search-2 \
+> docker run -it --rm --name redis-stack \
```

Comment:
As you're using the redis-stack container, perhaps you also want to expose redis insight?
```
-p 8081:8081
```